pkg/k8sutil/create.go

```diff
@@ -161,3 +166,67 @@ func parseJSONManifest(data []byte) ([]manifest, error) {
 	}
 	return manifests, nil
 }
+
+// UpdateNamespace updates the namespace.
+func UpdateNamespace(namespace Namespace, kubeconfig []byte) error {
+	cs, err := NewClientset(kubeconfig)
+	if err != nil {
+		return fmt.Errorf("creating clientset: %w", err)
+	}
+
+	if namespace.Name == "" {
+		return fmt.Errorf("namespace name can't be empty")
+	}
+
+	ns, err := cs.CoreV1().Namespaces().Get(context.TODO(), namespace.Name, metav1.GetOptions{})
```

Comment:
Maybe instead of taking `kubeconfig []byte` as a parameter, we could take https://godoc.org/k8s.io/client-go/kubernetes/typed/core/v1#NamespaceInterface? This would make testing and the function itself simpler.